index.html

```diff
@@ -21,15 +21,19 @@
       <header>
         <div class="header">
           <div class="logo"><a href="#">My Logo</a></div>
+
           <nav>
+            <span class="bars" id="barsBtn"
+              ><a href="#"><i class="fa-solid fa-bars"></i
+            ></a></span>
+
             <div class="menu-content">
               <ul>
                 <li><a href="#">Portfolio</a></li>
                 <li><a href="#">About</a></li>
                 <li><a href="#">Contact</a></li>
```

Comment:
- [ ] **Required:** You have done an amazing job. 👏 
However, the nav links when clicked should take you to the section on the webpage.

_**Hint**_ 
- Kindly listen for a click event on the links.
- And take the user to the appropriate section

OR

- Give the sections `id` attributes if they don't have any.
- pass the value to the links.
- [See this.](https://webplatform.github.io/docs/guides/html_links/)